garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -161,9 +161,9 @@ end
 concommand.Add("ttt_force_detective", force_detective)
 
 
-local function force_spectate(ply, cmd, arg)
+local function force_spectate(ply, arg)

```

Comment:
The concommand can't work this way.
The callback has the cmd as the second argument. So the concommand for backwards compatibility wouldn't work.